source/indexes/compound-index.txt

```diff
@@ -0,0 +1,99 @@
+.. _cpp-compound-index:
+
+================
+Compound Indexes
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: index, query, optimization, efficiency
+
+Overview
+--------
+
+**Compound indexes** are indexes that hold references to multiple
+fields within a collection's documents. These indexes improve multi-field query and
+sort performance.
+
+To create a compound index, call the ``create_index()`` method and specify a document containing: 
```

Comment:
S: We try to use full sentences when [introducing lists](https://www.mongodb.com/docs/meta/style-guide/style/lists/#introductory-text), so you can reword to:
```suggestion
To create a compound index, call the ``create_index()`` method and specify a document containing the following information: 
```